pages/about/code-of-conduct.vue

```diff
@@ -38,20 +38,97 @@
                     >
                         {{ description }}
                     </p>
-                    <ol v-if="rule.list" class="list-decimal ml-5">
-                        <li
-                            v-for="(li, liIndex) in rule.list"
-                            :key="`coc-list-${liIndex}`"
-                        >
-                            {{ li }}
-                        </li>
-                    </ol>
-                    <p
-                        v-if="rule.addition"
-                        class="text-xs md:text-base leading-5 md:leading-8"
+                    <!-- 三大準則圖片 -->
+                    <div
+                        v-if="i == 0"
+                        class="
+                            flex
+                            sm:justify-around
+                            justify-between
+                            mx-2
+                            my-1
+                            relative
+                        "
                     >
-                        {{ rule.addition }}
-                    </p>
+                        <div v-for="(img, j) in sub1" :key="`sub_img_${j}`">
+                            <div
+                                class="
+                                    w-16
+                                    sm:w-20
+                                    h-36
+                                    sm:h-40
+                                    bg-contain bg-no-repeat
+                                    mx-2
+                                    md:mx-5
+                                    my-10
+                                    sm:my-5
+                                    flex
+                                    justify-center
+                                "
+                                :style="getSub1Style(img)"
+                            >
+                                <span class="img_text">
+                                    {{ rule.list[j] }}
+                                </span>
+                            </div>
+                        </div>
+                    </div>
+                    <div
+                        v-for="(subtitle, subtitleIndex) in rule.subtitle"
+                        :key="`coc-description-${subtitleIndex}`"
```

Comment:
**issue (blocking):** It would cause duplicated keys warning with LINE 36. 
![image](https://user-images.githubusercontent.com/18432820/223051592-1e99d9c1-a033-4dfa-9381-b9f8f1eb3537.png)

Reply:
ＯＫ